ethereum/sidecar/attestation_validation_test.go

```diff
@@ -0,0 +1,227 @@
+		},
+		{
+			name:        "ConfirmedUnlocks is confirmed",
+			attestation: defaultAttestation(),
+			pre: func(tav *testAttestationValidation) {
+				tav.mockBridgeContract.EXPECT().
+					ValidateAssetsUnlocked(gomock.Any()).
+					Return(true, nil).
+					Times(1)
+				tav.mockBridgeContract.EXPECT().
+					ConfirmedUnlocks(gomock.Any()).
+					Return(true, nil).
+					Times(1)
+			},
+		},
+		{
+			name:        "Attestation failed with error",
+			attestation: defaultAttestation(),
+			pre: func(tav *testAttestationValidation) {
+				tav.mockBridgeContract.EXPECT().
+					ValidateAssetsUnlocked(gomock.Any()).
+					Return(true, nil).
+					Times(1)
+				tav.mockBridgeContract.EXPECT().
+					ConfirmedUnlocks(gomock.Any()).
+					Return(false, nil).
+					Times(1)
+				expectedError := errors.New("network error")
+				tav.mockBridgeContract.EXPECT().
+					Attestations(gomock.Any()).
+					Return(big.NewInt(0), expectedError).
+					Times(1)
+			},
+			expectErr: "network error",
+		},
+		{
+			name:        "ValidatorIDs failed with error",
+			attestation: defaultAttestation(),
+			pre: func(tav *testAttestationValidation) {
+				tav.mockBridgeContract.EXPECT().
+					ValidateAssetsUnlocked(gomock.Any()).
+					Return(true, nil).
+					Times(1)
+				tav.mockBridgeContract.EXPECT().
+					ConfirmedUnlocks(gomock.Any()).
+					Return(false, nil).
+					Times(1)
+				tav.mockBridgeContract.EXPECT().
+					Attestations(gomock.Any()).
+					Return(big.NewInt(0), nil).
+					Times(1)
+				expectedError := errors.New("network error")
+				tav.mockBridgeContract.EXPECT().
+					ValidatorIDs(gomock.Any()).
+					Return(uint8(10), expectedError).
+					Times(1)
+			},
+			expectErr: "network error",
+		},
+		{
+			name:        "Validator not in the bitmap",
+			attestation: defaultAttestation(),
+			pre: func(tav *testAttestationValidation) {
+				tav.mockBridgeContract.EXPECT().
+					ValidateAssetsUnlocked(gomock.Any()).
+					Return(true, nil).
+					Times(1)
+				tav.mockBridgeContract.EXPECT().
+					ConfirmedUnlocks(gomock.Any()).
+					Return(false, nil).
+					Times(1)
+				tav.mockBridgeContract.EXPECT().
+					Attestations(gomock.Any()).
+					Return(new(big.Int).SetBit(big.NewInt(1), int(10), 1), nil).
+					Times(1)
+				tav.mockBridgeContract.EXPECT().
+					ValidatorIDs(gomock.Any()).
+					Return(uint8(10), nil).
+					Times(1)
+			},
+			expectErr: ErrValidatorNotInTheBitmap.Error(),
+		},
+		{
+			name:        "Validation succeeded, nothing to do",
+			attestation: defaultAttestation(),
+			pre: func(tav *testAttestationValidation) {
+				tav.mockBridgeContract.EXPECT().
+					ValidateAssetsUnlocked(gomock.Any()).
+					Return(true, nil).
+					Times(1)
+				tav.mockBridgeContract.EXPECT().
+					ConfirmedUnlocks(gomock.Any()).
+					Return(false, nil).
+					Times(1)
+				tav.mockBridgeContract.EXPECT().
+					Attestations(gomock.Any()).
+					Return(new(big.Int).SetBit(big.NewInt(1), int(10), 0), nil).
```

Comment:
Just as in the previous comment, shouldn't it be:
```suggestion
					Return(new(big.Int).SetBit(big.NewInt(0), int(10), 1), nil).
```

Reply:
a6b3cb227524057d08d360350bb01cafd839c7d2